### PR TITLE
[Vendor] Update go-ldap to v3.2.4

### PR DESCRIPTION
**Pull request by 6543**
`go get -u` had an issue with it, this fix it :)

### CODE REVIEW

**Review by techknowlogick — COMMENTED** [review]
Because it is v3 you'll need to reference `github.com/go-ldap/ldap/v3`

**Comment by 6543** [comment]
@techknowlogick somehow missed that ... now it works as it should :)

File: modules/auth/ldap/ldap.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 
 	"code.gitea.io/gitea/modules/log"
 
-	"gopkg.in/ldap.v3"
+	"github.com/go-ldap/ldap/v3"
```
Comment (zeripath) [inline modules/auth/ldap/ldap.go:17]: Is this deliberate?

Comment (6543) [reply modules/auth/ldap/ldap.go:17]: yes

Comment (6543) [reply modules/auth/ldap/ldap.go:17]: @zeripath https://github.com/go-ldap/ldap/blob/master/go.mod#L1

**Review by zeripath — APPROVED** [review]

**Review by techknowlogick — COMMENTED** [review]
> somehow missed that ...

no worries. I just made this same upgrade in another project so I was recently aware of it.

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**